src/PolyhedralGeometry/Polyhedron/johnson.jl

```diff
@@ -0,0 +1,2427 @@
+function _johnson_solid(::Val{9})
+  NF, sr5 = quadratic_field(5)
+  NFy, y = NF["y"]
+  MF, mr = number_field(y^2 - (5 + sr5)//10, "a")
+  EMF, mre = Hecke.embedded_field(MF, real_embeddings(MF)[4])
+  sre5 = EMF(sr5)
+  V = [1//2 -mre*(sre5+1)//4 1//2;
+       1//2 -mre*(sre5+1)//4 -1//2;
+       -1//2 -mre*(sre5+1)//4 1//2;
+       -1//2 -mre*(sre5+1)//4 -1//2;
+       (1+sre5)//4 mre*(sre5-1)//4 1//2;
+       (1+sre5)//4 mre*(sre5-1)//4 -1//2;
+       -(1+sre5)//4 mre*(sre5-1)//4 1//2;
+       -(1+sre5)//4 mre*(sre5-1)//4 -1//2;
+       0 mre 1//2;
+       0 mre -1//2;
+       0 0 (1+mre*(sre5-1))//2]
+  res = convex_hull(EMF, V; non_redundant = true)
+  Polymake.setdescription!(pm_object(res), "Johnson solid J9: Elongated pentagonal pyramid")
+  return res
+end
+
+function _johnson_solid(::Val{10})
+  Qx, x = QQ["x"]
+  NF, qr8 = number_field(x^4 - 8, "a")
+  ENF, qre8 = Hecke.embedded_field(NF, real_embeddings(NF)[2])
+  sre2 = (qre8^2)//2
+  V = [0 0 (2*sre2 + qre8)//4;
+       1//2 1//2 qre8//4;
+       1//2 -1//2 qre8//4;
+       -1//2 1//2 qre8//4;
+       -1//2 -1//2 qre8//4;
+       0 sre2//2 -qre8//4;
+       0 -sre2//2 -qre8//4;
+       sre2//2 0 -qre8//4;
+       -sre2//2 0 -qre8//4]
```

Comment:
If I see it right, this is a 2427 lines file which is essentially a "database" consisting of polyhedra given by coordinates over some number fields.

Wouldn't this be a perfect application for @antonydellavecchia's serialization code? I.e. have a data file instead of all this code which needs to be parsed, compiled and optimized for nothing really...